src/bin.ts

```diff
@@ -48,7 +48,7 @@ export function main(
   const state: BootstrapState = {
     shouldUseChildProcess: false,
     isInChildProcess: false,
-    entrypoint: __filename,
+    tsNodeScript: __filename,
```

Comment:
Was this renamed to indicate that this is the path to ts-node's bootstrapper, but *not* the user's desired entrypoint?

Reply:
Yes, I renamed that mostly to avoid the ambiguity here (at least ambiguity to me). Happy to revert if you would want it the old way.

Reply:
Renaming is good, makes sense to me.